src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,88 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final int GROWTH_FACTOR = 2;
+    private static final float LOAD_FACTOR = 0.75f;
+    private Node<K,V>[] nodes;
+    private int size;
+    private int currentCapacity;
+
+    public MyHashMap() {
+        nodes = new Node[DEFAULT_CAPACITY];
+        currentCapacity = DEFAULT_CAPACITY;
+    }
 
     @Override
     public void put(K key, V value) {
-
+        Node<K, V> newNode = new Node<>(key, value);
+        int threshold = (int) (currentCapacity * LOAD_FACTOR);
```

Comment:
You should leave the `threshold` as it was (a private field), then in constructor set the default value, and then change the value only in the `resize` method.